modules/host/storageobligations.go

```diff
@@ -574,8 +574,15 @@ func (h *Host) removeStorageObligation(so storageObligation, sos storageObligati
 		}
 	}
 	if sos == obligationSucceeded {
+		// Empty obligations don't submit a storage proof. The revenue for an empty
+		// storage obligation should equal the contract cost of the obligation
+		if len(so.SectorRoots) == 0 {
+			h.log.Printf("Not submitted a storage proof for empty contract. Revenue is %v.\n", so.ContractCost.Add(so.PotentialStorageRevenue).Add(so.PotentialDownloadRevenue).Add(so.PotentialUploadRevenue))
+		} else {
+			h.log.Printf("Successfully submitted a storage proof. Revenue is %v.\n", so.ContractCost.Add(so.PotentialStorageRevenue).Add(so.PotentialDownloadRevenue).Add(so.PotentialUploadRevenue))
+		}
```

Comment:
Since the revenue is calculated the same way for both `Printf` statements we can simplify that to make it a bit more readable.
```
revenue := so.ContractCost.Add(so.PotentialStorageRevenue).Add(so.PotentialDownloadRevenue).Add(so.PotentialUploadRevenue)
if len(so.SectorRoots) == 0 { 
    h.log.Printf("No need to submit a storage proof for an empty contract. Revenue is %v.\n", revenue)
} else {
    h.log.Printf("Successfully submitted a storage proof. Revenue is %v.\n", revenue)
}
```